src/main/java/com/future/function/qa/pages/core/StickyNotes/EditStickyNotesPage.java

```diff
@@ -0,0 +1,9 @@
+package com.future.function.qa.pages.core.StickyNotes;
+
+import com.future.function.qa.model.UrlConstant;
+import com.future.function.qa.pages.base.MenuPage;
+import net.thucydides.core.annotations.DefaultUrl;
+
+@DefaultUrl(UrlConstant.EDIT_STICKY_NOTES_URL)
+public class EditStickyNotesPage extends MenuPage {
+}
```

Comment:
please add more elements in this page if decided to add edit action automation on sticky notes

Reply:
done